src/index.js

```diff
@@ -0,0 +1,130 @@
+import 'regenerator-runtime/runtime';
+import axios from 'axios';
+
+const state = {
+    temperature: 40
+};
+
+const tempText = document.querySelector("#tempValue");
+const landscapeText = document.querySelector("#landscape");
+const cityText = document.querySelector("#cityNameInput");
+const skyText = document.querySelector("#sky");
+
+const changeSky = (sky) => {
+    if (sky === "sunny") {
+        skyText.innerHTML = "☁️ ☁️ ☁️ ☀️ ☁️ ☁️"
+    } else if (sky === "cloudy") {
+        skyText.innerHTML = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️"
+    } else if (sky === "rainy") {
+        skyText.innerHTML = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧"
+    } else if (sky === "snowy") {
+        skyText.innerHTML = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨"
+    }
+};
+
+const changeSkyValue = () => {
+    const skyValue = document.getElementById('skySelect').value;
+    changeSky(skyValue);
+};
+
+const increaseTemp = () => {
+    state.temperature += 1;
+    updateTempUi();
+};
+
+const updateTempUi = () => {
+    tempText.innerHTML = `${state.temperature}`;
```

Comment:
You don't need to use string templating here. You'd use string templating for concatenating strings, like:

```javascript
const baseUrl = "localhost:500";
axios.get(`${baseUrl}/cats`)
```